packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/testProfileRunOperations/testProfileRunOperationsClient.ts

```diff
@@ -9,15 +9,15 @@ import {
   DeleteTestProfileRunOptionalParams,
   GetTestProfileRunOptionalParams,
   ListTestProfileRunsOptionalParams,
-  StopOptionalParams,
+  StopTestProfileRunOptionalParams,
```

Comment:
do you merge to the latest main? i believe this should be updated already during our upgrading pr.
https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/testProfileRunOperations/testProfileRunOperationsClient.ts#L12

Reply:
Updated now.